pkg/asyncdr/asyncdr.go

```diff
@@ -295,10 +296,11 @@ func CreateSchedulePolicy(policyName string, interval int) (pol *storkapi.Schedu
 }
 
 // WaitForNumOfMigration waits for a certain number of migrations to complete.
-func WaitForNumOfMigration(schedName string, schedNamespace string, count int, miginterval int) (map[string]string, error) {
+func WaitForNumOfMigration(schedName string, schedNamespace string, count int, miginterval int) (map[string]string, []map[string]string, error) {
 	migInterval := time.Minute * time.Duration(miginterval)
 	migTimeout := time.Minute * time.Duration(count*miginterval)
 	expectedMigrations := make(map[string]string)
+	var migschedulestats []map[string]string
```

Comment:
How about appending keys to a single map rather than creating a list of maps?

Reply:
This wont work as they will be separate maps for seperate migrations. Multiple migrations created as a part of schedule and they disappears as next become successful, so as soon as one become successful we have to collect data in map for individual.

Reply:
Oh okay. Thanks!